### DISCUSSION TITLE
Game won't start or crashes

### DISCUSSION THREAD

**Discussion by DaraceAk** [ErinaSugino/Starbound-SBR-Mod#150] [Trouble Shooting] [answered]
Hello, I have a problem with the mod, I want to use the SBR mod, however I have problems with this one, here is my mod folder:

Monsterhormones
SBR aphrodites bow patches
sexbound-reborn-1.2_r2-packed
lewdbound0.7.1.pak
mods go here
sexbound-aphrodites-bow-6.2.0.pak
sexbound-defeat-2.7.0.pak
SxB teratodefeat 0.4.6.pak
SxB teratophilia 0.4.8.pak
SxB temtems teratoscene.pak

when I have all that, my game crashes on launch "Fatal exception caught: (applicationexeption) Application thre exceptionduring startup" and puts a ton of weird lines after, when I remove the Lewdbown, Teratodefeat and sexbound_defeat files, the game launches, but crashes when I take my character, so I tested with sexbound-reborn-1.2_r2-sourced (without the 3 files) and the game crashes when the game logo is displayed. and with classic sexbound, without SxB temtems teratoscene.pak it seems to work. Obviously my mods are in the right place. I believe that, logically, I followed everything that was said on the instructions on the "troubleshooting guide" page, I don't rule out having messed up on this point, but it seems to me that I did everything right. How could I make SBR work? Thanks in advance

![Capture d'écran 2024-12-29 174051](https://github.com/user-attachments/assets/dcd9376c-1a81-4975-9619-da82b90aa533)

**Comment by ErinaSugino** [comment, marked as answer]
You have, in fact, *not* followed the troubleshooting guide, as you do not have Sexbound installed properly. Which crashes your game cause Sexbound is not installed, and needed for your other mods.
